underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystemFactory.java

```diff
@@ -46,7 +46,7 @@ public HdfsUnderFileSystemFactory() {}
 
   @Override
   public UnderFileSystem create(String path, Configuration configuration, Object conf) {
-    Preconditions.checkArgument(path != null, "path may not be null");
+    Preconditions.checkArgument(path != null);
```

Comment:
put `"cannot create a file system for a null path"` into a constant in `PreconditionMessage`?
It is still friendly if we could have an error message